python3/ltk/actions/init_action.py

```diff
@@ -462,10 +459,8 @@ def set_workflow(self, community_id, project_id):
             # End Python 3
             try:
                 if choice == '':
-                    for x in workflow_info:
-                        if x == workflow_id:
-                            workflow_id, workflow_name = x, 'Project Default'
-                    logger.info('\nSelected "{0}" {1}.\n'.format(workflow_name, 'workflow'))
+                    workflow_id, workflow_name = 'Project Default', 'Project Default'
+                    logger.info('\nKept "{0}" {1}.\n'.format(workflow_name, 'workflow'))
                     return workflow_id, False 
                 else:
                     choice = int(choice)
```

Comment:
We also don't seem to be taking into account the case where the user inputs the choice for `Project Default`.

Reply:
The only thing this code was doing was matching `Project Default` to the `workflow_id` we get from `get_project`. `False` indicates whether the project should be updated or not.